j2cli/cli.py

```diff
@@ -190,6 +190,7 @@ def render_command(cwd, environ, stdin, argv):
     renderer.register_filters({
         'docker_link': filters.docker_link,
         'env': filters.env,
+        'bool': bool,
```

Comment:
Can you also update the docs/README with this PR?  

You should update docs anytime you change code; especially when changing a public interface. This PR seems very low-effort.